src/Uno.UI/UI/Xaml/Application.macOS.cs

```diff
@@ -76,7 +78,17 @@ private ApplicationTheme GetDefaultSystemTheme()
 			else
 			{
 				return ApplicationTheme.Dark;
-			}			
+			}
+		}
+
+		private void SetCurrentLanguage()
+		{
+			try
+			{
+				var language = NSLocale.PreferredLanguages[0];
+				CultureInfo.CurrentUICulture = CultureInfo.CreateSpecificCulture(language);
+			}
+			catch (Exception) { }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Handle the exception or explain in a comment why it can be ignored.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5521872)

Reply:
@jeromelaban what should we do in this case? I really don't care if it crash since it can if the language cannot be translated to a `CultureInfo`. 

Will a comment be sufficient here?